assets/scss/seasons/3_harryette_mullen.scss

```diff
@@ -0,0 +1,63 @@
+.mullen {
+  padding-bottom: initial;
+  background-color: #224fc7;
+  color: white;
+  font-family: 'Times';
+  align-items: center;
+  margin-top: var(--margin);
+
+
+
+  .layout-wrapper {
+    display: flex;
+    width: 462px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    margin: auto;
+
+
+    @include mobile {
+      width: 415px;
+      margin: auto;
```

Comment:
no need to dupe here if defined above.

Reply:
heya im tryign t o change the mobile font size since the text doesnt fit but the code in root overrites mine.. trying to resolve this but if i cant sending the code as is forward